File: back_viajes/src/main/java/com/softtek/Modelo/AmadeusDatos.java
package com.softtek.Modelo;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class AmadeusDatos {

    private String originLocationCode;
    private String destinationLocationCode;
    private String departureDate;
    private String returnDate;
    private int adults;
    private boolean nonStop;
    private double precioViaje;

    private int ratings;

    private String nombre_hotel;
    private String id_hotel;
    private double latitud;
    private double lengitud;
    private double precioHotel;

    private String actividad1;
    private String actividad2;
    private String actividad3;
    private Double precio_actividades;
}

File: front_viajes/src/app/_modelo/AmadeusDatos.ts
export interface AmadeusDatos {
  originLocationCode: string;
  destinationLocationCode: string;
  departureDate: string;
  returnDate: string;
  adults: number;
  nonStop: boolean;
  precioViaje: number;
  
  ratings: number;

  nombre_hotel: string;
  id_hotel: string;
  latitud: string;
  lengitud: string;
  precioHotel: number;

  actividad1: string;
  actividad2: string;
  actividad3: string;
  precio_actividades: number;
  maletas: number;
}
